core/deployment/src/main/java/io/quarkus/deployment/pkg/steps/JarResultBuildStep.java

```diff
@@ -784,21 +787,24 @@ private void doLegacyThinJarGeneration(CurateOutcomeBuildItem curateOutcomeBuild
         final Map<String, List<byte[]>> services = new HashMap<>();
 
         final List<AppDependency> appDeps = curateOutcomeBuildItem.getEffectiveModel().getUserDependencies();
+        final Set<String> finalIgnoredEntries = new HashSet<>(IGNORED_ENTRIES);
+        packageConfig.userConfiguredIgnoredEntries.ifPresent(finalIgnoredEntries::addAll);
```

Comment:
Aren't we supporting ignored entries only for the uber jar packaging?

Reply:
Yes, I added support for thin JARs too (when the resource is inside the project). This may be useful for files that should be ignored while creating the JAR